### DISCUSSION TITLE
valheim server on vps

### DISCUSSION THREAD

**Discussion by Madumtv** [Nimdy/Dedicated_Valheim_Server_Script#305] [Q&A]
i would like to launch valheim on vps.
i can putt on my vps any claasic os i want.
i have just a question: i need to launch valheim and a game on the vps or i need just to config and start a server.
Do you know if in the case i need to launch valheim on vps like a player, i need 2 account? one for the srv and one for playing? 
or it's like a minecraft server and i can launch a serv on vps and connect on it via my pc in classic mode? 
thanks for informations !

**Comment by DonaldValentine** [comment]
Setting up a Valheim server on a VPS is a great way to enjoy smooth gameplay with friends anytime. Using a reliable [vps starter](https://arzhost.com/vps/) plan can give you enough power and stability without breaking the bank. Just pick a VPS with good CPU and RAM, install the Valheim dedicated server, and you’re ready to host your own world 24/7. It’s perfect if you want more control and better performance than a regular home server.
